jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/codegeneration/MethodChainHandler.java

```diff
@@ -0,0 +1,34 @@
+package us.abstracta.jmeter.javadsl.codegeneration;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MethodChainHandler {
```

Comment:
I think original code is simpler (less lines of codes) and this class is not worth it to only encapsulate a list and provide a string builder for it, since this logic is only used once and is simple enough.